processing/data_collection/gazette/spiders/sc_blumenau.py

```diff
@@ -0,0 +1,56 @@
+
+from dateparser import parse
+from datetime import datetime
+from scrapy import Request
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class ScBlumenauSpider(BaseGazetteSpider):
+    TERRITORY_ID = '4202404'
+    NEXT_PAGE_URL = 'https://www.diariomunicipal.sc.gov.br{}'
+    POSSIBLE_GAZETTE_CSS = '.quiet'
+    NEXT_PAGE_CSS = '.pagination li.next:not(.disabled) a::attr(href)'
+
+    allowed_domains = ['diariomunicipal.sc.gov.br']
+    name = 'sc_blumenau'
+    start_urls = ['https://www.diariomunicipal.sc.gov.br/site/?r=site/index&q=cod_municipio%3A164']
+
+    def parse(self, response):
+        """
+        @url https://www.diariomunicipal.sc.gov.br/site/?r=site/index&q=cod_municipio%3A164
+        @returns requests 1
+        @scrapes date file_urls is_extra_edition territory_id power scraped_at
+        """
+
+        possible_gazettes = response.css(self.POSSIBLE_GAZETTE_CSS);
+        for element in possible_gazettes:
+            url = self.extract_url(element)
```

Comment:
Why create a class method that is used only once and have only one line?
`url = element.css('a::attr(href)').extract_first()` is clear enough on its intent and make it easier to maintain later (don't need to go over the code to understand what is happening)